packages/manager/modules/billing/src/orders/purchaseOrders/billing-orders-purchases.routing.js

```diff
@@ -61,7 +63,7 @@ export default /* @ngInject */ ($stateProvider) => {
 
       hideBreadcrumb: /* @ngInject */ () => false,
 
-      minDate: /* @ngInject */ () => new Date(),
+      minDate: /* @ngInject */ () => moment().format('YYYY-MM-DD'),
```

Comment:
'YYYY-MM-DD' is used in multiple places. Can you extract it as a constant.

Reply:
It's applied